src/app/demo/demo.component.html

```diff
@@ -209,11 +209,46 @@
   </div>
 
   <div class="fullscreen">
-    <div>
+    <nav class="navbar navbar-expand flex-column flex-md-row flex-wrap bd-navbar mb-4">
+      <a class="navbar-brand" href="#">
+          <img src="https://angular.io/assets/images/logos/angular/angular.svg" width="32px" height="32px"/>
+          @madoar/angular-archwizard
+      </a>
```

Comment:
This link looks redundant to me? I mean the existing nav bar on the left already contains a button in its header (with the text: `angular-archwizard`) which leads to the main page of the demo application.

Reply:
done.